backend/python/app/services/interfaces/story_service.py

```diff
@@ -50,3 +50,14 @@ def get_story_translations(self, user_id, translate):
         :rtype: list of StoryTranslationResponseDTO's
         """
         pass
+
+    @abstractmethod
+    def get_story_available_for_review(self, level, language):
+        """
+        Return a list of stories available to be reviewed by user
+        :param level: level of user 
+        language: language being searched for
```

Comment:
nit: `language: language being searched for` -> `param language: language being searched for`